core/common/src/main/java/alluxio/cli/AbstractShell.java

```diff
@@ -39,18 +40,22 @@
   private static final Logger LOG = LoggerFactory.getLogger(AbstractShell.class);
 
   private Map<String, String[]> mCommandAlias;
+  private Set<String> mUnstableAlias;
   private Map<String, Command> mCommands;
   protected InstancedConfiguration mConfiguration;
 
   /**
    * Creates a new instance of {@link AbstractShell}.
    *
    * @param commandAlias replacements for commands
+   * @param unstableAlias set of unstable aliases which may be removed in the future
    * @param conf Alluxio configuration
    */
-  public AbstractShell(Map<String, String[]> commandAlias, InstancedConfiguration conf) {
+  public AbstractShell(Map<String, String[]> commandAlias,
```

Comment:
I actually feel this constructor is not ideal --- there are three or four subclasses and most of them just passing two `null`s, meaning this constructor is very specialized only to make `FilesystemShell` work.

Instead of having the static private member  `CMD_ALIAS` and `UNSTABLE_ALIAS`, can we define alias in each Command and add something like `getAliases` into the `Command` interface?  The current way looks less clean to me (passing two alias sets).

If we are running out of time, I am cool with the current approach for preview.